stubs/Pillow/PIL/Image.pyi

```diff
@@ -163,7 +163,9 @@ class Image:
     def getprojection(self) -> tuple[list[int], list[int]]: ...
     def histogram(self, mask: Image | None = ..., extrema: tuple[int, int] | tuple[float, float] | None = ...) -> list[int]: ...
     def entropy(self, mask: Image | None = ..., extrema: tuple[int, int] | tuple[float, float] | None = ...) -> float: ...
-    def paste(self, im: Image, box: tuple[float, float] | _Box | None = ..., mask: Image | None = ...) -> None: ...
+    def paste(
+        self, im: Image | tuple[int, int] | int, box: tuple[float, float] | _Box | None = ..., mask: Image | None = ...
+    ) -> None: ...
```

Comment:
One thing I noticed when I just tested this is that the `box` argument must be given when `im` is an `int` (and I assume a tuple). We could express this using overloads, but I'm also fine accepting this as is for now.